icons/vr-headset.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20 6a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2h-4a2 2 0 0 1-1.6-.8l-1.6-2.13a1 1 0 0 0-1.6 0L9.6 17.2A2 2 0 0 1 8 18H4a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2z" />
+  <circle cx="17" cy="12" r="1" />
+  <circle cx="7" cy="12" r="1" />
```

Comment:
Those two will break in some cases such as in figma, r needs to be at least 2px to not violate 2px gap rule.


```suggestion
```
What about calling it `rectangle-goggles` or something, that way we don't have the guideline violation, and it's more generic, and it's implied that it is based on the `rectangle` icon.
<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M20+6a2+2+0+0+1+2+2v8a2+2+0+0+1-2+2h-4a2+2+0+0+1-1.6-.8l-1.6-2.13a1+1+0+0+0-1.6+0L9.6+17.2A2+2+0+0+1+8+18H4a2+2+0+0+1-2-2V8a2+2+0+0+1+2-2z%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMjAgNmEyIDIgMCAwIDEgMiAydjhhMiAyIDAgMCAxLTIgMmgtNGEyIDIgMCAwIDEtMS42LS44bC0xLjYtMi4xM2ExIDEgMCAwIDAtMS42IDBMOS42IDE3LjJBMiAyIDAgMCAxIDggMThINGEyIDIgMCAwIDEtMi0yVjhhMiAyIDAgMCAxIDItMnoiIC8+Cjwvc3ZnPg==.svg"/><br/>Open lucide studio</a>

Reply:
I just went ahead and threw this together which I think might work better.

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M+15.5+12+L+18+12%22+/%3E%3Cpath+d=%22M+6+12+L+8.5+12%22+/%3E%3Cpath+d=%22M20+6a2+2+0+0+1+2+2v8a2+2+0+0+1-2+2h-4a2+2+0+0+1-1.6-.8l-1.6-2.13a1+1+0+0+0-1.6+0L9.6+17.2A2+2+0+0+1+8+18H4a2+2+0+0+1-2-2V8a2+2+0+0+1+2-2z%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNIDE1LjUgMTIgTCAxOCAxMiIgLz4KICA8cGF0aCBkPSJNIDYgMTIgTCA4LjUgMTIiIC8+CiAgPHBhdGggZD0iTTIwIDZhMiAyIDAgMCAxIDIgMnY4YTIgMiAwIDAgMS0yIDJoLTRhMiAyIDAgMCAxLTEuNi0uOGwtMS42LTIuMTNhMSAxIDAgMCAwLTEuNiAwTDkuNiAxNy4yQTIgMiAwIDAgMSA4IDE4SDRhMiAyIDAgMCAxLTItMlY4YTIgMiAwIDAgMSAyLTJ6IiAvPgo8L3N2Zz4=.svg"/><br/>Open lucide studio</a>

Reply:
Now it looks sleepy, I would really just keep it stupid simple and only do the outline.

Reply:
aight at least I tried lol, I'll go ahead and commit your changes rq